DistFiles/reference_texts/Russian/russian.glyssen

```diff
@@ -9,26 +9,27 @@
     <numerals />
     <fontFamily>Charis SIL</fontFamily>
     <fontSizeInPoints>14</fontSizeInPoints>
-	<heSaidText>сказал.</heSaidText>
+    <heSaidText>сказал.</heSaidText>
+    <reportingClause>сказал.</reportingClause>
   </language>
   <identification>
     <name>Synodal Translation (1876)</name>
     <nameLocal />
-    <systemId type="fcbhOT"></systemId>
-    <systemId type="fcbhNT">106</systemId>
+    <systemId type="fcbhOT" />
```

Comment:
not sure if this is okay?

Reply:
Yes, it's okay. The XML serializer defaults to this shorter syntax for empty elements. I thought about reverting it since it's not a meaningful change. But it's harmless.